package/yast2-firewall.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jul 28 11:01:02 UTC 2020 - Michal Filka <mfilka@suse.com>
+
+- AutoYaST: moved the configuration into first stage
```

Comment:
Miss bug number entry, I created a bug to be used by all the modules which moved the configuration to the first stage